src/containers/Pomodoro/Pomodoro.js

```diff
@@ -10,16 +12,22 @@ const TimerBox = styled.div`
   text-align: center;
 `;
 
-const StartButton = styled.button``;
+const Button = styled.button`
+  width: 45px;
+  height: 45px;
+  border-radius: 50%;
+  border-style: none;
+`;
 
 class Pomodoro extends Component {
   state = {
     timerIsRunning: false,
-    seconds: 3
+    seconds: 5,
+    sessionEndSound: Sound.status.STOPPED
   };
 
   handleStartButton = () => {
-    this.setState({ timerIsRunning: true });
+    this.setState({ timerIsRunning: true, sessionEndSound: Sound.status.STOPPED });
```

Comment:
pls rename the state to what it actually represents, in this case it's `soundStatus` :)